recipes/kaiju/meta.yaml

```diff
@@ -6,10 +6,10 @@ package:
 
 source:
   url: https://github.com/bioinformatics-centre/kaiju/archive/v{{ version }}.tar.gz
-  sha256: 45de5d879a836aaf8f1547af95f592ab4c2ac71d8ac62eff7e3fa4f49772ac97
+  sha256: dfb755016fc99838c251b8adf4f5a7f12feef1be12dbdaee6595d837ec1dcf44
```

Comment:
Can you create a new version instead? We should usually not change the source during a build-number bump. Every change in the source should ideally get a new version. This is, for example, because we backup the tarballs.

Reply:
hm, in principle I could make a new version, but the old v1.8.0.tar.gz is gone for good already anyways.